src/partisan_hyparview_peer_service_manager.erl

```diff
@@ -462,7 +478,9 @@ handle_info({'EXIT', From, _Reason},
                           connections=Connections}};
 
 handle_info({connected, Peer, Tag, _RemoteState}, State) ->
-    handle_connect(Peer, Tag, State);
+    %% @todo retreive the PeerEpoch from the Peer.
```

Comment:
Can't you just expand the tuple like we did when we added ```Tag```?

(remote state is going away as soon as I have time to remove it.)

Can you change the message to `{connected, Peer, Tag, Epoch, RemoteState}`?